src/views/Activate/Syncs/__tests__/SyncRunsTableItem.test.tsx

```diff
@@ -0,0 +1,95 @@
+import { render, screen } from '@testing-library/react';
+import '@testing-library/jest-dom';
+import { TableItem } from '@/views/Activate/Syncs/SyncRuns/SyncRunTableItem';
+import { SyncRunsResponse } from '../types';
+
+describe('TableItem', () => {
+  const mockSyncRunsData: SyncRunsResponse[] = [
+    {
+      id: '1',
+      type: 'sync_runs',
+      attributes: {
+        sync_id: '1',
+        status: 'success',
+        source_id: '2',
+        destination_id: '3',
+        started_at: '2024-03-15T06:26:00.345Z',
+        finished_at: '2024-03-15T06:26:07.374Z',
+        created_at: '2024-03-15T06:26:00.299Z',
+        updated_at: '2024-03-15T06:26:07.378Z',
+        duration: 1.0,
+        total_query_rows: 500,
+        total_rows: 500,
+        successful_rows: 500,
+        failed_rows: 0,
+        error: null,
+      },
+    },
+    {
+      id: '2',
+      type: 'sync_runs',
+      attributes: {
+        sync_id: '1',
+        status: 'failed',
+        source_id: '2',
+        destination_id: '3',
+        started_at: '2024-03-15T07:26:00.345Z',
+        finished_at: '2024-03-15T07:26:07.374Z',
+        created_at: '2024-03-15T07:26:00.299Z',
+        updated_at: '2024-03-15T07:26:07.378Z',
+        duration: 1.0,
+        total_query_rows: 500,
+        total_rows: 500,
+        successful_rows: 450,
+        failed_rows: 50,
+        error: null,
+      },
+    },
+  ];
+
+  it('should render start time correctly', () => {
+    render(<TableItem field='start_time' data={mockSyncRunsData[0]} />);
+    expect(screen.getByText('01/01/2020 at 05:01 am'));
```

Comment:
this is wrong, it should be something like

```suggestion
    expect(screen.getByText('01/01/2020 at 05:01 am')).toBeTruthy();
```
[Ref](https://jestjs.io/docs/expect#tobetruthy)

the way you have written, it will always pass the test 😅

Reply:
Here, the problem is that we did get the text - but now what? we want to test that the text is visible. Which can be by using these truthy values.
you can read more [here](https://jestjs.io/docs/expect)